client/bio-commandline/src/main/kotlin/uk/ac/ebi/biostd/client/cli/commands/ValidateFileListCommand.kt

```diff
@@ -3,24 +3,31 @@ package uk.ac.ebi.biostd.client.cli.commands
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.required
-import uk.ac.ebi.biostd.client.cli.common.CommonParameters
-import uk.ac.ebi.biostd.client.cli.common.SubmissionParameters
+import uk.ac.ebi.biostd.client.cli.common.CommonParameters.ON_BEHALF_HELP
+import uk.ac.ebi.biostd.client.cli.common.CommonParameters.PASSWORD_HELP
+import uk.ac.ebi.biostd.client.cli.common.CommonParameters.SERVER_HELP
+import uk.ac.ebi.biostd.client.cli.common.CommonParameters.USER_HELP
+import uk.ac.ebi.biostd.client.cli.common.FileListValidationParameters.PREVIOUS_VERSION
+import uk.ac.ebi.biostd.client.cli.common.FileListValidationParameters.ROOT_PATH
+import uk.ac.ebi.biostd.client.cli.common.SubmissionParameters.FILE_LIST_PATH
 import uk.ac.ebi.biostd.client.cli.dto.SecurityConfig
 import uk.ac.ebi.biostd.client.cli.dto.ValidateFileListRequest
 import uk.ac.ebi.biostd.client.cli.services.SubmissionService
 
 internal class ValidateFileListCommand(
     private val submissionService: SubmissionService
 ) : CliktCommand(name = "validateFileList") {
-    private val server by option("-s", "--server", help = CommonParameters.SERVER_HELP).required()
-    private val user by option("-u", "--user", help = CommonParameters.USER_HELP).required()
-    private val password by option("-p", "--password", help = CommonParameters.PASSWORD_HELP).required()
-    private val onBehalf by option("-b", "--onBehalf", help = CommonParameters.ON_BEHALF_HELP)
-    private val fileListPath by option("-f", "--fileListPath", help = SubmissionParameters.FILE_LIST_PATH).required()
+    private val server by option("-s", "--server", help = SERVER_HELP).required()
+    private val user by option("-u", "--user", help = USER_HELP).required()
+    private val password by option("-p", "--password", help = PASSWORD_HELP).required()
+    private val onBehalf by option("-b", "--onBehalf", help = ON_BEHALF_HELP)
+    private val fileListPath by option("-f", "--fileListPath", help = FILE_LIST_PATH).required()
+    private val previousVersion by option("-pv", "--previousVersion", help = PREVIOUS_VERSION)
```

Comment:
I think this (and http parameter) should be only accNo, which means we are validation file list inside an specific submission.